### ISSUE TITLE
Remaining non-generated HWIntrinsic tests should be moved to be generated

### ISSUE THREAD

**Issue by tannergooding** [dotnet/coreclr#19246]
Currently, there are a number of HWIntrinsic tests that are hand written which leads to maintainability issues and gaps in coverage as compared to the other HWIntrinsic tests (templates have been updated a few times, in order to fix bugs, or add new scenarios, which impacts the vast majority of other HWIntrinsic tests).

It would be beneficial to move them to the same code generation templates as the other HWIntrinsic tests.

**Comment by tannergooding** [comment]
NOTE: SSE2 is being tracked separately, as it is a bulkier work item: https://github.com/dotnet/coreclr/issues/19245

**Comment by tannergooding** [comment]
CC. @fiigii, @CarolEidt, @eerhardt

**Comment by 4creators** [comment]
I can handle this after finishing #19245

**Comment by BruceForstall** [comment]
@tannergooding Is there any work left to do here?

@4creators Do you still want this assigned to you?

**Comment by tannergooding** [comment]
Yes. There are still a number (but now a minority) of tests that should be templated rather than manually written.

**Comment by BruceForstall** [comment]
@tannergooding Is this just a dup of https://github.com/dotnet/coreclr/issues/19400?

**Comment by tannergooding** [comment]
I think #19400 is likely a dupe of this, since this one was created first; but yes 😄 

Looking at the list in #19400, it isn't in sync with some of the tests that have been ported over to be templated now.
